modules/cluster-and-ha-management/pages/expand-a-cluster.adoc

```diff
@@ -94,6 +94,32 @@ If you choose to supply a staging location, make sure that the TigerGraph Linux
 For example, assume you are expanding from a 6-node cluster with a replication factor of 2 and a partitioning factor of 3, to a 10-node cluster with a replication factor of 2 and a partitioning factor of 5, and the size of the data root folder on a node is 50GB.
 You would need more than `(1 + ceiling(3/5)) * 50) = 100 GB` of free space on the staging path.
 
+==== Config region aware
+
+If you need to enable xref:region-aware.adoc[region-aware], the format of `node_ip_list` needs to include region information.
+
+[tabs]
+====
+Enable Region Aware::
++
+--
+[source,console]
+----
+$ gadmin cluster expand us-east-1:m3:192.168.1.3,us-east-2:m4:192.168.1.4 --region-aware
+----
+--
+Disable Region Aware::
++
+--
+[source,console]
+----
+$ gadmin cluster expand m3:192.168.1.3,m4:192.168.1.4 --region-aware=false
+----
+--
+====
+
+If you do not need to configure region aware, you do not need to specify `region-aware` option, and the option will be consistent with the current cluster.
```

Comment:
- Let's give a warning that cross region will have extended latency for queries.
- We should also mention zones together with regions. Specifically, we should replay all the region with region/zone. Zones are typically closer geographically and actually preferable to be used. See more discussion here: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html